src/components/scale-button.js

```diff
@@ -169,4 +162,30 @@ AFRAME.registerComponent("scale-button", {
     this.objectMatrix.scale(this.deltaScale);
     setMatrixWorld(this.objectToScale, this.objectMatrix);
   }
+}
+
+AFRAME.registerComponent("scale-button", {
+  init() {
+    this.handler = new ScalingHandler(this.el.object3D, this.el.sceneEl.systems["transform-selected-object"]);
```

Comment:
I'm not familiar with A-Frame component/system lifecycles. Is it safe to assume that `transform-selected-object` system is already registered here?

Reply:
I think it should per:
https://github.com/mozilla/hubs/blob/3c897c0461fca5591646ce531af0a76c2cef3f32/src/hub.js#L123-L124

Reply:
Thanks.

And if I'm right, system seems to be registered to `sceneEl.systems` when `AFRAME.registerSystem` is called (when a file is imported) while `Component.init()` seems to be called later (when the component first initialized?). So probably it is safe to assume here.